packages/app/src/features/liquidity-flow/hooks/use-token-info.ts

```diff
@@ -12,14 +12,14 @@ export function useTokenInfo(tokenName: SupportedToken): {
   const { address } = tokenContracts[tokenName]
   const marketData = useMarketData()
   const { discount = 0 } = marketData?.[address] ?? {}
-  const priceInUsd = usePriceInUsd()[tokenName]
+  const priceInUsd = usePriceInUsd()?.[tokenName]
```

Comment:
```suggestion
  const priceInUsd = usePriceInUsd()?.[tokenName] ?? 0
```
and we don't need to update line 22.

The reason and the solution are one one line.

Reply:
Thx, edited